### PR TITLE
Charts api

### PR DESCRIPTION
**Pull request by Wild-Monkey-Coder**
The most basic step of implementing the charts API. Now loads a chart on the webpage based on hard coded data from the JS file.

### CODE REVIEW

File: portfolio/src/main/webapp/MainPage.html [hunk 1]
```diff
@@ -138,6 +142,17 @@ <h1>Interests</h1>
...
         </div>
+        <div class="section-header">
+            <h1>Projects - STEP &amp; Other</h1>
+        </div>
+        <div class="sections" id="projects">
+            <div class="project-container">
+                <div class="project-text">
+                    <p>This is the charts API project I implemented.<p>
+                </div>
+                <button onclick="redirectToChartsPage()" class="project-button">View Project</button>
```
Comment (AlexanderEllis) [inline portfolio/src/main/webapp/MainPage.html:153]: Any reason you didn't make this a simple `a` element link?

File: portfolio/src/main/webapp/ChartProject.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Charts</title>
+        <script src="https://www.gstatic.com/charts/loader.js"></script>
+        <script src="chartsScript.js"></script>
+    </head>
+    <body>
+        <div id="games-on-steam-chart">
```
Comment (AlexanderEllis) [inline portfolio/src/main/webapp/ChartProject.html:9]: Mind adding a screenshot in the description for this new page with the drawn chart?

File: portfolio/src/main/webapp/chartsScript.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+/**
+ * Creates a basic chart and displays it on the page.
+ **/
+function drawChart() {
+    var data = new google.visualization.DataTable();
+    data.addColumn('string', 'Task');
+    data.addColumn('number', 'Hours per Day');
+    data.addRows([
+        ['Dont Starve Together', 146],
```
Comment (AlexanderEllis) [inline portfolio/src/main/webapp/chartsScript.js:27]: This is the hardest game I think I've ever played

File: portfolio/src/main/webapp/MainPage.html (hunk 1 above)
Comment (Wild-Monkey-Coder) [reply portfolio/src/main/webapp/MainPage.html:153]: I hadn't thought of using an <a> element link, it was just what made sense in my mind at the time to me based on how I wanted the site to look and the ability to style a button in the future.

File: portfolio/src/main/webapp/chartsScript.js (hunk 3 above)
Comment (Wild-Monkey-Coder) [reply portfolio/src/main/webapp/chartsScript.js:27]: Glad to meet a fellow DST gamer! I agree, getting through winter the first time was quite the experience.

File: portfolio/src/main/webapp/ChartProject.html (hunk 2 above)
Comment (Wild-Monkey-Coder) [reply portfolio/src/main/webapp/ChartProject.html:9]: image
Screenshot 2020-06-15 at 12 13 59 PM

**Pull request merged**